README.md

```diff
@@ -99,23 +98,30 @@ cmake --build . --target install
 ```
 TODO: Add simple client and server source code
 
-### Example Programs
-If the `-DBUILD_EXAMPLES=ON` CMake option is used during building,
-example client and server executables will be built and placed
-in the `${CMAKE_BINARY_DIR}/bin` directory.
-Example configuration data is also provided in the `examples/data`
-directory.
+### Running a Test Client and Server Handshake
+If the `-DBUILD_TOOL=ON` CMake option is used during building,
+the XTT tool will be built and placed in the `${CMAKE_BINARY_DIR}/tool` directory.
+
+To create configuration data, run:  
```

Comment:
This is just server configuration, right? So maybe to be clear we should say "to create server configuration data"

Reply:
Or, to be explicit, maybe separate the root creation from the server creation, something like:
```
To create root configuration data, run:
xtt genkey -v root_priv.bin -b root_pub.bin
xtt genrootcert

Then, to create server configuration data under that root, run:
xtt genkey -v server_priv.bin -b server_pub.bin
xtt genservercert
```